gallery/gallery/lib/pages/demo.dart

```diff
@@ -245,6 +247,7 @@ class _DemoPageState extends State<DemoPage> with TickerProviderStateMixin {
                 _state == _DemoState.fullscreen ? selectedIconColor : iconColor,
             onPressed: () => _handleTap(_DemoState.fullscreen),
           ),
+        SizedBox(width: appBarPadding),
```

Comment:
This would only solve the problem for the buttons on the right, but the back button on the left would still have the problem.

Reply:
Hence the SizedBox on the left :)

Reply:
Oh I see what you mean

Reply:
It looks like the back button is already afforded enough space for the ripple to show